src/content/docs/infrastructure/infrastructure-monitoring/get-started/choose-infra-install-method.mdx

```diff
@@ -0,0 +1,89 @@
+    - Get started
+translate:
+    - jp
+    - kr
+metaDescription: An overview of installation methods for New Relic's infrastructure agent and infrastructure integrations.
+redirects:
+    - /docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic
+    - /docs/infrastructure/new-relic-infrastructure/installation
+    - /docs/infrastructure/install-infrastructure-agent
+    - /docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent
+freshnessValidatedDate: never
+---
+
+import osAnsibleRed from 'images/os_icon_ansible-red.webp'
+
+import osChef from 'images/os_icon_chef.webp'
+
+import osEbs from 'images/os_icon_ebs.webp'
+
+import osPuppet from 'images/os_icon_puppet.webp'
+
+import infrastructureAngetsandNr from 'images/infrastructure_diagram_angets-and-nr.webp'
+
+import osDocker from 'images/os_icon_docker.webp'
+
+The infrastructure agent is an executable file that collects data about your hosts. You can enable the infrastructure agent to report data from [some third party services](/docs/infrastructure/host-integrations/installation/install-infrastructure-host-integrations) and to collect [log data](/docs/logs/enable-log-monitoring-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent). 
+
+The infrastructure agent can run on Windows, macOS, and many Linux distributions. This document describes the different methods you can use to install the infrastructure agent. To get started, you'll need a [New Relic account](https://newrelic.com/signup) and a license key.
+
+## Install the infrastructure agent manually [#manual-install]
+
+We recommend following our manual install procedures if you're looking to monitor a couple hosts, but nothing more complex. In general, you'll use the command line to create a directory where the agent lives in, then add your license key information. Depending on your operating system, you may need to create and edit a config file as well. To install manually, start here:
+
+* [Install on Linux systems with a package manager](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux)
+* [Install on macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos)
+* Install on Windows with [MSI install](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows), [zip install](/docs/infrastructure/install-infrastructure-agent/windows-installation/zip-assisted-install-infrastructure-agent-windows), or [manual zip install](/docs/infrastructure/install-infrastructure-agent/windows-installation/zip-manual-install-infrastructure-agent-windows).
+
+## Deploy the infrastructure agent programmatically [#deploy-programmatically]
+
+You can deploy the infrastructure agent programmatically with several config management and deploy tools:
```

Comment:
I think programmatically is the wrong word here, I'd recommend trying to simplify it